src/main/resources/cmmn/Generiek_zaakafhandelmodel.cmmn.xml

```diff
@@ -66,6 +66,12 @@
                     <extensionElements>
                         <modeler:flowable-idm-initiator xmlns:modeler="http://flowable.org/modeler">
                             <![CDATA[true]]></modeler:flowable-idm-initiator>
+                        <flowable:planItemLifecycleListener sourceState="enabled" targetState="active"
+                                                            class="net.atos.zac.flowable.cmmn.UpdateZaakLifecycleListener">
+                            <flowable:field name="status">
+                                <flowable:string><![CDATA[Aanvullende informatie vereist]]></flowable:string>
```

Comment:
Should we have full text here? Or should we use a code instead, so that the translation into a proper description can be done in the UI using the standard pattern ?

Reply:
This has nothing to do with the UI. This is a link to our `UpdateZaakLifecycleListener` which will update the zaak with this status. Yes, I think using codes will be better. However this is just a spike so will not be done here.

Reply:
I added your remark to: https://dimpact.atlassian.net/wiki/spaces/PZW/pages/165543954/Proposal+to+add+zaakstatus+Aanvullende+informatie+vereist